backend/src/services/super-admin/super-admin-service.ts

```diff
@@ -99,7 +99,12 @@ export const superAdminServiceFactory = ({
     await orgService.createOrganization(userInfo.user.id, userInfo.user.email, initialOrganizationName);
 
     await updateServerCfg({ initialized: true });
-    const token = await authService.generateUserTokens(userInfo.user, ip, userAgent);
+    const token = await authService.generateUserTokens({
+      user: userInfo.user,
+      ip,
+      userAgent,
+      organizationId: undefined
```

Comment:
This isn't required because the initial super-user organization is not one that requires org-level auth scoping like a SAML org.

I guess it can optionally be set but it can also be `undefined` since the nature of that initial organization is that it doesn't have SAML configured.